src/main/java/service/VehicleService.java

```diff
@@ -42,6 +42,7 @@ public Vehicle fetchById(int id) {
     @Override
     @CacheEvict(value="vehicle", key="#id")
     public void delete(int id) throws Exception {
+        //TODO finish implementation of delete to allow the following line to work
```

Comment:
Code Review Input # 1 - Comment changes
To keep consistency with most other comments in this application, instead of just commenting out a line of code, you should give context as to why it needed to be left out.
Added TODO description.

References:
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3
https://www.baeldung.com/java-clean-code

Reply:
I wouldn't add a TODO as part of a code review... If there's something to be done here, complete it.